platform-controller/src/main/java/org/hobbit/controller/tools/ProjectSearcher.java

```diff
@@ -16,6 +16,7 @@ public static void main(String[] args) throws InterruptedException {
         if (args.length < 2) {
             System.out.println(
                     "Error: wrong usage. The following parameters are necessary:\n <security-token> <search-term(s)>");
```

Comment:
Actually, the whole tools package is not covered. Can it be fixed?

Reply:
See comment above.